basics_scenarios/sitewise_scenario/SPECIFICATION.md

```diff
@@ -0,0 +1,336 @@
+# AWS IoT SiteWise Service Scenario Specification
+
+## Overview
+This SDK Basics scenario demonstrates how to interact with AWS IoT SiteWise using an AWS SDK. 
+It demonstrates various tasks such as creating a SiteWise Asset Model, creating an asset, 
+sending data to the asset, reading data, and so on.  Finally this scenario demonstrates how 
+to clean up resources. Its purpose is to demonstrate how to get up and running with 
+AWS IoT SiteWise and an AWS SDK.
+
+## Resources
+This Basics scenario requires an IAM role that has permissions to work with AWS IoT 
+SiteWise service. The scenario creates this resource using a CloudFormation template.
+
+## Hello AWS IoT SiteWise
+This program is intended for users not familiar with the AWS IoT SiteWise Service to easily get up and running. The logic is to show use of 
+`listAssetsPaginator`.
+
+## Basics Scenario Program Flow
+The AWS IoT SiteWise Basics scenario executes the following operations.
+
+1. **Create an AWS SiteWise Asset Model**:
+   - Description: This step creates an AWS SiteWise Asset Model by invoking the `createAssetModel` method.
+   - Exception Handling: Check to see if a `ResourceAlreadyExistsException` is thrown. 
+     If it is thrown, get the asset model ID and move on.
+
+2. **Create an AWS IoT SiteWise Asset**:
+   - Description: This operation creates an AWS SiteWise asset.
+   - The method `createAsset` is called to obtain the asset ID.
+   - Exception Handling: Check to see if a `ResourceNotFoundException` is thrown. If so, 
```

Comment:
A `ResourceNotFoundException` seems an odd Exception for a create* operation. 

A `[ResourceAlreadyExistsException](https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/iotsitewise/model/ResourceAlreadyExistsException.html) make more sense.

Reply:
Nice catch - it is a ResourceNotFoundException.

Reply:
ResourceALre4adyExists not applicable for this service operation. It will make dup asset names.

Reply:
Fixed